developer_guide/installation/system_requirements/system_install_osx_1011.rst

```diff
@@ -0,0 +1,276 @@
+* Work with MySQL 5.7. MySQL 5.7 is not officially supported but works in experimental mode if you disable the ONLY_FULL_GROUP_BY mode.
+
+Download and install the Mysql Community edition server from the official website: https://dev.mysql.com/downloads/mysql/
+
+Select your Mac OS X platform then download and install the DMG Archive. This package installs a preference pane extension for Mysql.
+
+.. note::
+
+    The root password will be handed to you during the installation process. Don't forget to copy it in order to change it after the installation completes.
+    The Mysql files will be installed in the following path `/usr/local/mysql`.
+
+Run the Mysql server on your machine by starting it from the preferences.
+
+Check that you can connect to mysql with:
+
+.. code-block:: bash
+
+        $ mysql -u root -p # Enter the copied password
+
+.. note::
+
+    During the first connection, Mysql will ask you to change the root password for security reasons, follow the indicated steps.
+
+PHP installation
+****************
+
+Since Akeneo PIM 1.6, the minimal PHP version is PHP 5.6.
+You have two possibilities:
+
+* Currently, the only supported version of PHP for Akeneo PIM is 5.6.
+
+.. _php56:
+
+PHP 5.6 (supported)
+^^^^^^^^^^^^^^^^^^^
+
+* In order to install specific php versions (5.6, 7.0 and even 7.1), add those repositories:
+
+.. code-block:: bash
+
+    $ brew tap homebrew/dupes
+    $ brew tap homebrew/versions
+    $ brew tap homebrew/homebrew-php
+
+* You can now install PHP 5.6 and the needed libraries:
+
+.. code-block:: bash
+
+    $ brew update
+    $ brew install php56 --with-httpd24
+    $ brew install php56-apcu php56-mcrypt php56-opcache php56-intl
+
+.. note::
+
+    The `--with-httpd24` option installs the package httpd24 (apache) automatically and compiles the libphp5.so extension for apache. If you want to
+    use the OS X built-in apache version, you need to unlink the package using the following command: `brew unlink httpd24` (This command does not delete
+    the libphp5.so extension that you will need to include your apache configuration).
+
+* Check that PHP 5.6 is now your current PHP version with:
+
+.. code-block:: bash
+
+    $ php -v
+
+* Now you can directly continue by :ref:`choosing_product_storage`.
+
+Choosing the product storage
+****************************
+
+.. include:: /reference/technical_information/choose_database.rst.inc
+
+Based on this formula, either you need a :ref:`mongodb-install-1604`, either you can directly go to the :ref:`system-configuration-1604` section.
+
+.. _mongodb-install-1604:
+
+MongoDB Installation (optional)
+*******************************
+
+PHP 5.6 (supported)
+^^^^^^^^^^^^^^^^^^^
+
+* Install MongoDB server and PHP driver
+
+.. note::
+
+    Akeneo PIM will not work with MongoDB 3.*. *The supported versions are 2.4 and 2.6*.
+
+.. code-block:: bash
+
+    $ brew update
+    $ brew install mongodb24
```

Comment:
We need to rework this mongodb installation to `curl & tar & ln -s` from the mongo website to install the 2.4